runners/spark/src/main/java/org/apache/beam/runners/spark/SparkPipelineRunner.java

```diff
@@ -268,12 +447,30 @@ public static void main(String[] args) throws Exception {
   }
 
   private static class SparkPipelineRunnerConfiguration {
+    @Option(
+        name = "--event-log-enabled",
+        usage = "Set it to true if event logs should be saved to Spark History Server directory")
+    private String eventLogEnabled = "false";
+
+    String getEventLogEnabled() {
+      return this.eventLogEnabled;
+    }
+
     @Option(
         name = "--base-job-name",
         usage =
             "The job to run. This must correspond to a subdirectory of the jar's BEAM-PIPELINE "
                 + "directory. *Only needs to be specified if the jar contains multiple pipelines.*")
     private String baseJobName = null;
+
+    @Option(
+        name = "--spark-history-dir",
+        usage = "Spark history dir to store logs (e.g. /tmp/spark-events/)")
+    private String sparkHistoryDir = "/tmp/spark-events/";
```

Comment:
Please remove the `--spark-history-dir` option from `SparkPipelineRunnerConfiguration` since it is unused and unneeded.

Reply:
Done.